AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -5468,6 +5470,313 @@ function New-AzDnsForwarder {
 #region DFS-N cmdlets
 #endregion
 
+#region Share level permissions migration cmdlets
+function Get-StorageAccountDetails {

```

Comment:
Is this supposed to be used by customers?
The name is a "Get-*" operation on an account but has a side effect of creating a share object, which is misleading. Need a better name. If this is just for testing purpose, I am not sure if we need this cmdlet.

Reply:
Removed this function. As you are saying it doing more than the name suggests